### PR TITLE
Improving ActionLink Component #36496

### PR DESCRIPTION
**Pull request by zocoster**
### Prerequisites

Please make sure you can check the following two boxes:

- [X] I have read the **CONTRIBUTING** document
- [X] My code follows the code style of this project

### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [X] Bug fix (non-breaking change which fixes an issue, please reference the issue id)
- [ ] New feature (non-breaking change which adds functionality, make sure to open an associated issue first)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My change requires a change to the documentation
- [X] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [X] All new and existing tests passed

### Description

Changed ActionLink font-weight from bold to 500.
Fixed disabled font-color to pinkSwan.
Added different color when it's focused.

It's my first PR, be nice ;)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Precise UI Changelog
 
+## 0.5.1
```
Comment (FlorianRappl) [inline CHANGELOG.md]: No. We have not released 0.5.0.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "precise-ui",
-  "version": "0.5.0",
+  "version": "0.5.1",
```
Comment (FlorianRappl) [inline package.json]: No, really...

File: src/common.ts [hunk 3]
```diff
@@ -303,6 +303,10 @@ export interface PreciseFullTheme extends PreciseThemeColors {
    * Background color of hovered action link.
    */
   actionLinkHoverBackground: string;
+  /**
+   * Background color of focused action link.
+   */
+  actionLinkFocusBackground: string;
```
Comment (FlorianRappl) [inline src/common.ts]: Please remove this one - we are exclusively using global settings in here.

File: src/components/ActionLink/Example.md [hunk 4]
```diff
@@ -1,11 +1,19 @@
-A simple call to action link:
+A simple call to Action Link:
```
Comment (FlorianRappl) [inline src/components/ActionLink/Example.md]: Its either `ActionLink` (note formatting) or "an action link".

File: src/components/ActionLink/Example.md [hunk 5]
```diff
@@ -1,11 +1,19 @@
...
 
 ```jsx
 const { ActionLink } = require('precise-ui');
 
 <ActionLink href="#🍕">Click me</ActionLink>
 ```
 
+This is a disabled Action Link:
```
Comment (FlorianRappl) [inline src/components/ActionLink/Example.md]: See above.

File: src/themes.ts [hunk 6]
```diff
@@ -125,8 +125,9 @@ export const light: PreciseFullTheme = {
   accordionLine: `1px solid ${colors.lighterGray}`,
   accordionPadding: `${distance.medium}`,
   accordionContentPadding: `${distance.small} ${distance.medium} ${distance.xlarge} ${remCalc('50px')}`,
-  actionLinkHoverBackground: colors.skyBlue,
-  actionLinkDisabledText: colors.white,
+  actionLinkHoverBackground: colors.ocean,
+  actionLinkFocusBackground: colors.midnight,
```
Comment (FlorianRappl) [inline src/themes.ts]: midnight is ui7 - ocean and pinkSwan I don't know. afaik they are not on the theme; so either this is wrongly specified in the pattern library or missing from the color spec.

Comment (zocoster) [reply src/themes.ts]: ui6 was changed from transparent to ocean
we looked for all appearances of ui6 inside pharos and renamed it to transparent
we removed actionLinkHoverBackground and actionLinkFocusBackground from themes

**Pull request merged**